oca_dependencies.txt

```diff
@@ -1 +1,3 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
```

Comment:
you can trash this file: [MQT=PIP](https://github.com/OCA/stock-logistics-workflow/blob/14.0/.travis.yml#L30)

Reply:
hi @simahawk , thx for your review. These changes have been cherry-picked to this MR